src/pages/SignUp/SignUp.js

```diff
@@ -1,7 +1,154 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import INPUT_LIST from './InputList';
+import SignUpInput from './SignUpInput';
+import './SignUp.scss';
 
-const SignUp = () => {
-  return <div />;
-};
+function SignUp() {
+  const [inputs, setInputs] = useState({
+    name: '',
+    id: '',
+    pw: '',
+    birth: '',
+    phone: '',
+    gender: '',
+    email: '',
+    recommend: '',
+  });
+
+  function handleChangeState(e) {
+    setInputs({ ...inputs, [e.target.name]: e.target.value });
```

Comment:
e.target 구조분해할당해서 사용하면 좀 더 깔끔해지겠네요!

Reply:
밑에 부분에 구조분해 할당해서 했는데 어떤 부분을 더 변경해야될지 잘 모르겠습니다... 구조분해할당에 대해서 더 공부 후 변경해보겠습니다~!

Reply:
```suggestion
    const { name, value } = e.target;
    setInputs({ ...inputs, [name]: value });
```
위와 같은 변경 말씀드린거였습니다 확인 해보시고 반영하시면 될 것 같아요 :)